packages/chat-widget/src/ui/components.tsx

```diff
@@ -475,6 +475,12 @@ export const PinBubble: React.FunctionComponent<{
   </PinBubbleContainer>
 );
 
+PinBubble.propTypes = {
```

Comment:
@michaelglass what is the rationale behind prop types?

Reply:
added a less intrusive fix for what was a lint failure.